src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/CreateSiteFilteringFiles.java

```diff
@@ -1,8 +1,8 @@
 package org.broadinstitute.hellbender.tools.gvs.filtering;
 
 import htsjdk.variant.variantcontext.VariantContext;
-import io.grpc.netty.shaded.io.netty.util.internal.StringUtil;
 
+import io.netty.util.internal.StringUtil;
```

Comment:
why is this change needed? or is it part of one of the upgrades?

Reply:
We were importing the "shaded" version of this before which isn't really the library, but gradle's repackaging of it and it's not stable.  This is the public version.